seq2seq/evaluator/evaluator.py

```diff
@@ -114,7 +114,10 @@ def evaluate(self, model, data, get_batch_data, ponderer, **get_batch_kwargs):
         for batch in batch_iterator:
             input_variable, input_lengths, target_variable = get_batch_data(batch, **get_batch_kwargs)
 
-            decoder_outputs, decoder_hidden, other = model(input_variable, input_lengths.tolist(), target_variable['decoder_output'])
+            # If attentive guidance is provided in the data set, add this as kwarg
+            kwargs = {} if 'provided_attention' not in target_variable else {'provided_attention': target_variable['provided_attention']}
+
+            decoder_outputs, decoder_hidden, other = model(input_variable, input_lengths.tolist(), target_variable['decoder_output'], **kwargs)
```

Comment:
if 'provided_attention' is already in target_variable, can we not just pass it with the target_variable so that we don't need the keyword argument? Are the kwargs used for anything else?

Reply:
In `Seq2Seq.forward()` we also add the `input_length`. 
We could indeed just pass the entire `target_variables` dict to the model, we would just need to change the signature then of `DecoderRNN.forward()`. Either we unpack the dictionary within `Seq2seq.forward()` and feed the decoder the separate variables as separate argument, or we pass the entire `target_variable` dict to the decoder where it should be unpacked into the separate variables.